### PR TITLE
Add GetAppId,uTerminateBackgroundApplication desc

### PR DESCRIPTION
**Pull request by gotoextreme**
### Change Description ###
Add GetAppId,uTerminateBackgroundApplication desc

### CODE REVIEW

File: docs/application/dotnet/guides/app-management/app-manager.md [hunk 1]
```diff
@@ -40,6 +40,12 @@ To get the application running context and its details, and to operate on the co
 
 1.  Get the context of the currently-running application by creating an instance of the [Tizen.Applications.ApplicationRunningContext](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.ApplicationRunningContext.html) class, with the ID of the application from which the context is being obtained as a parameter.
 
+    If you do not know your App ID, you can retrieve it as follows.
+
+    ```
+    string applicationId = ApplicationManager.GetAppId((Your PID);
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/app-management/app-manager.md]: Would you please check `((Your PID)`?
Also, I can't see GetAppId from https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.ApplicationManager.html. 

Is this update really for master branch, the branch for tizen 5.0?

Comment (gotoextreme) [reply docs/application/dotnet/guides/app-management/app-manager.md]: This update is for 5.5.
http://suprem.sec.samsung.net/jira/browse/TCSACR-202

Comment (mijin85cho) [reply docs/application/dotnet/guides/app-management/app-manager.md]: Would you please rebase this PR? I switched to tizen_5.5_prepare branch, which is for tizen 5.5.

Comment (gotoextreme) [reply docs/application/dotnet/guides/app-management/app-manager.md]: I'll make new PR based on tizen_5.5_prepare

**Comment by gotoextreme** [comment]
I'

**Pull request closed without merging**